ekip/nationalparks/models.py

```diff
@@ -115,6 +115,16 @@ class FederalSite(models.Model):
     def kids_pass(self):
         return (self.annual_pass and self.active_participant)
 
+    def save(self, *args, **kwargs):
+        """ Generate and save a slug when this object is saved for the first
+        time. """
+
+        if not self.id:
+            name = self.name
+            name = name.replace('National Wildlife Refuge', 'NWR')
+            self.slug = slugify(name + self.city)[:80]
```

Comment:
Just name and city don't generate decent enough slugs - especially if you're trying to replace the slugify management command. If you look at that, it does a lot of work to ensure the slugs are meaningful, short and unique.